docs/proposals/Avatar/README.md

```diff
@@ -0,0 +1,78 @@
+# Avatar
+
+An `<Avatar />` component will be used to display an identifier (image or
+initials) for a user.
+
+Rebecca Li has done some
+[great research](https://jobber.atlassian.net/browse/JOB-19028) on the usage of
+avatars in Jobber, as well as the potential usage of colors within avatars.
+
+## Interface
+
+```
+<Avatar
+    image="ImageUrl"
+    name="Eddy Sims"
+    initials="ES"
+    color="#8f2d56" | color="var(--color-red)"
+    size="large" | size={32}
+/>
+```
+
+## Props Table
+
+| name     | type   | default  | description                               |
+| -------- | ------ | -------- | ----------------------------------------- |
+| image    | String | -        | The avatar image to be use                |
+| name     | String | -        | A users name to be used for accessibility |
+| initials | String | -        | Initials to show if the image is null     |
+| color    | String | greyBlue | The border color that represents the user |
```

Comment:
We may want to be more specific than "string" here - is it a string that can be evaluated in CSS as a color?

Reply:
As well, I would suggest that the user's avatar container should receive the `color` prop as a background colour if there is no photo present

Reply:
Also of note, the border colour is white by default, with a greyBlue background

Reply:
@chris-at-jobber I have addressed this (in the component). But the default state is kind of weird, in that the background is greyBlue and the border is white. 

Any other state, the background/border would be the same color.